micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/pages/employee/ProductDetails/ProductDetailsComponentUpdated.js

```
@@ -0,0 +1,347 @@
+        .filter(c => c.type === "paragraph")
+        .map(c => `${t(c.text)}`);
+
+    const roleItems = config[0].subsections[0].content.filter(c => c.type === "role");
+
+    content.about.roles = roleItems.length > 0
+        ? roleItems.map(c => ({
+            description: t(c.text || "Data missing")
+        }))
+        : [
+            { description: t(`${module}` + "_SECTION1_ROLE_1") },
+            { description: t(`${module}` + "_SECTION1_ROLE_2") }
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)